src/analyzer/protocol/dns/events.bif

```diff
@@ -528,6 +528,52 @@ event dns_EDNS_addl%(c: connection, msg: dns_msg, ans: dns_edns_additional%);
 ##    dns_skip_all_addl dns_skip_all_auth dns_skip_auth
 event dns_EDNS_ecs%(c: connection, msg: dns_msg, opt: dns_edns_ecs%);
 
+## Generated for DNS replies of type *EDNS*. For replies with multiple options,
```

Comment:
This comment and the one below could be more descriptive about what type of EDNS query causes them to be generated.